python-main.js

```diff
@@ -1411,6 +1433,15 @@ function web_editor(config) {
         }
         return result;
     }
+    
+    function get_language_hash(){
```

Comment:
I think I would prefer to use URL query parameters (`?lang=en`) instead of URL fragments, can we update this function to use that instead? Or is there a good reason to use fragments instead?

Reply:
At the moment this code allows you to do both, it uses the get_qs_context function to read in all queries and looks for '?l=en' in the if statement from line 1396-1406. I used 'l' to be consistent with the other queries' formats of 'n', 'c', and 's'. My plan was to only keep one of the methods of searching the URL but didn't know which one you wanted.